tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -39,6 +39,11 @@ Verify Model Can Be Deployed Via UI
     Click Button    Create server
     Wait Until Page Contains    Data science projects
     Wait Until Page Contains Element    //button[.="Create data science project"]
+    ${already_present} =    Run Keyword And Return Status    Page Should Contain    ${PRJ_TITLE}
```

Comment:
if you want there is the `Project Should Be Listed`, nothing will change from functionality pov

Reply:
ty, need to do another pass on this kw and will fix this up then